src/pages/Game.tsx

```
@@ -0,0 +1,479 @@
+    return <div className="board">{cells}</div>
+  }
+
+  return (
+    <div className="game-page">
+      <div className="nav-bar">
+        <h1>ONITAMA</h1>
+        <div className="nav-links">
+          <Link to="/game">Game</Link>
+          <Link to="/how-to-play">How to Play</Link>
+          <Link to="/leaderboard">Leaderboard</Link>
+          <button onClick={onLogout}>Logout</button>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)